codespell_lib/data/dictionary.txt

```diff
@@ -7954,7 +7954,6 @@ shpae->shape
 shpaes->shapes
 shpapes->shapes
 shreak->shriek
-shrinked->shrunk
```

Comment:
Why? "shrinked" is nonstandard

Reply:
I was on the fence about it. If you think it should stay then I'll re-add

Reply:
Yeah let's put it back for now